# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/editing/run/delete.html.ini
# Commit: 554d63dad12e
# Full Hash: 554d63dad12e4d1f2560e99de1fafac9a97da026
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2023-09-27 09:02:41
# Regressor Bug: 1851951
# File Overlap Count: 1
# Description:
#   Bug 1851951 - Make `HTMLEditor` refer computed `display` instead of the HTML default style at considering block or inline element r=m_kato
#   
#   Blink [1] and WebKit [2] refers `display-outside` value when they consider whether an
#   element is a block or an inline.
#   
# ==============================================================================

diff -r 97dc7be3b93c -r 554d63dad12e testing/web-platform/meta/editing/run/delete.html.ini
--- a/testing/web-platform/meta/editing/run/delete.html.ini	Wed Sep 27 02:50:33 2023 +0000
+++ b/testing/web-platform/meta/editing/run/delete.html.ini	Wed Sep 27 03:01:26 2023 +0000
@@ -518,19 +518,24 @@
     expected: FAIL
 
   [[["delete",""\]\] "<p style=display:inline>fo[o<p style=display:inline>b\]ar" compare innerHTML]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [[["stylewithcss","true"\],["delete",""\]\] "<span style=display:block>fo[o</span><span style=display:block>b\]ar</span>" compare innerHTML]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [[["stylewithcss","false"\],["delete",""\]\] "<span style=display:block>fo[o</span><span style=display:block>b\]ar</span>" compare innerHTML]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [[["stylewithcss","true"\],["delete",""\]\] "<quasit style=display:block>fo[o</quasit><quasit style=display:block>b\]ar</quasit>" compare innerHTML]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [[["stylewithcss","false"\],["delete",""\]\] "<quasit style=display:block>fo[o</quasit><quasit style=display:block>b\]ar</quasit>" compare innerHTML]
-    expected: FAIL
+    expected:
+      if not early_beta_or_earlier: FAIL
 
   [[["delete",""\]\] "<ol><li>foo</ol>{}<br><ol><li>bar</ol>" compare innerHTML]
     expected: FAIL